tests/page/page-wait-for-response.spec.ts

```diff
@@ -113,3 +113,13 @@ it('should work with no timeout', async ({page, server}) => {
   ]);
   expect(response.url()).toBe(server.PREFIX + '/digits/2.png');
 });
+
+it('should resolve response.finished() 404 page', async ({ page, server, browserName }) => {
+  it.fixme(browserName === 'chromium');
+  await page.goto(server.EMPTY_PAGE);
+  const [response] = await Promise.all([
+    page.waitForResponse('**/404'),
+    page.evaluate(() => fetch('./404'))
```

Comment:
This should be `fetch('./404').then(r => r.text())`. Unless you fetch the body, the response never finishes because the browser just does not read from the network.

With that change, this test passes for me in all browsers.

Reply:
Thanks, @dgozman! That makes a lot of sense. I was incorrectly assuming the three browsers would behave the same with this test, but it looks like they have divergent behavior (but not a PW bug).